portfolio/src/main/webapp/script.js

```diff
@@ -49,14 +49,16 @@ function getRandomMessage() {
   });
 }
 
+//Fetches the comment from the server and adds it to the DOM.
 function fetchComments() {
-  fetch('/data').then(response => response.json()).then((comment) => {
+  fetch('/data').then(response => response.json()).then((comment) => {          //may have to change to .then(comments)
+      console.log("fetching comment" + comment);
     const commListElement = document.getElementById('comment-container');
     for (i = 0; i < comment.length; i++){
-        if (comment[i] == ""){continue;}
         commListElement.appendChild(createListElement(comment[i]));
     }
   });
+    console.log("adding comment to DOM");
```

Comment:
This log doesn't make much sense outside of the promise, because it will execute before anything is actually done